upload_artifacts.sh

```diff
@@ -1,21 +1,33 @@
 #!/bin/bash
 
-# Used in .travis.yml
+# This script is used in .travis.yml
+
+set -e
 
 BUCKET=piksi-buildroot-images
 PIKSI_VERSION=v3
+BUILD_VERSION=$(git rev-parse --short HEAD)
```

Comment:
Ah okay. I had just checked `git describe --tags` and that one said `fatal: No names found`